public/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>What's deployed? (React)</title>
```

Comment:
Let's not forget to remove the ` (React)` part.

Reply:
Oh definitely. This is here because during testing I've had both this and the prod version open at once and flip back and forth, and I kept getting confused which one I was looking at. :laughing:

Reply:
Smart!